### DISCUSSION TITLE
AIDente 2.1.2 Works on MBP 13 9.2 (mid 2012)

### DISCUSSION THREAD

**Discussion by saffuria** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#246] [General]
Thanks a lot!
I once burnt the small MagSafe charging board on my Mbp, blocking the central contact to stop charging.
Fortunately, I found a replacement on the bay, and God bless Ifixit.
Thank you again. :)

Stefano

**Comment by MatthiasKerbl** [comment]
Hi @saffuria,
I read about this method of blocking a contact. I think AlDente is a little bit more convenient. :)
You are welcome!

Matthias
